invenio_app_ils/ext.py

```diff
@@ -189,7 +189,8 @@ def init_app(self, app):
             Blueprint(
                 "invenio_app_ils_mail",
                 __name__,
-                template_folder="templates"
+                static_folder="static",
```

Comment:
I think it might be better not to touch mail for such thing. Can't we add it to the `invenio-theme` blueprint instead? Otherwise I would create one specific to ILS...

Reply:
I have created a dummy blue print only for static.

```python
app.register_blueprint(
    Blueprint(
        "invenio_app_ils_static",
        __name__,
        static_folder="static",
    )
)
```

This is the global static folder configuration, and the image can be accessed like `/static/images/placeholder.png`. We can always extend the `invenio-theme`, this will move the file from the global static to the theme module specific static, so it will be available from `theme/static/images/placeholder.png`. The later approach is the clean-separated approach, but the current one will suffice too.